### PR TITLE
Update getting started guide

### PR DESCRIPTION
**Pull request by ebrattli**

### CODE REVIEW

File: docs/source/cognite.rst [hunk 1]
```diff
@@ -13,50 +13,63 @@ You can also pass your API key directly to the CogniteClient.
...
-    >>> c = CogniteClient(api_key="your-key")
+    >>> c = CogniteClient(api_key="<your-api-key>", client_name="your-client-name")
 
 Instantiate a new client
 ------------------------
 Use this code to instantiate a client and get your login status. CDF returns an object with
-attributes that describe which project and service account your API key belongs to.
+attributes that describe which project and service account your API key belongs to. The client_name
+is used to give a unique identifier to the client. You can choose any string for the client_name. The client_name
+can also be provided through the :code: `COGNITE_CLIENT_NAME` environment variable.
```
Comment (erlendvollset) [inline docs/source/cognite.rst]: Legg til her "All examples in the documentation assume that the `COGNITE_CLIENT_NAME` is set.", og så droppe å sende det med i konstruktøren. Da slipper vi å oppdatere all docstring eksemplene også.

File: docs/source/cognite.rst [hunk 2]
```diff
@@ -13,50 +13,63 @@ You can also pass your API key directly to the CogniteClient.
...
 methods you can use to plot the data.
 
 .. code:: python
 
     >>> from cognite.client import CogniteClient
-    >>> c = CogniteClient()
-    >>> my_datapoints = c.datapoints.retrieve_multiple(
-    ...                     id=[1, 2],
+    >>> c = CogniteClient(client_name="your-client-name")
+    >>> my_datapoints = c.datapoints.retrieve_dataframe(
```
Comment (erlendvollset) [inline docs/source/cognite.rst]: This should be just `.retrieve(...)`.

`.retrieve_dataframe(...)` returns a pandas DataFrame, which does indeed have a `.plot()` method. But we want to show the `.plot()` method on the Datapoints/DatapointsList objects here.

**Comment by erlendvollset** [comment]
adding @arnekv as reviewer here to do a quick check on language.

File: docs/source/cognite.rst [hunk 3]
```diff
@@ -28,25 +31,36 @@ attributes that describe which project and service account your API key belongs
 
 Read more about the `CogniteClient`_ and the functionality it exposes below.
 
+Discovering time series
+-----------------------
+For the next examples, you will need to supply ids for the time series that you want to retrieve. This can be done by
```
Comment (arnekv) [inline docs/source/cognite.rst]: Replace "This can be done by..." with "You can do this by..."

File: docs/source/cognite.rst [hunk 4]
```diff
@@ -28,25 +31,36 @@ attributes that describe which project and service account your API key belongs
...
+    >>> from cognite.client import CogniteClient
+    >>> c = CogniteClient()
+    >>> c.time_series.list(include_metadata=False, limit=2)
+    
 Plot time series
 ----------------
 There are several ways of plotting a time series you have fetched from the API. The easiest is to call
 :code:`.plot()` on the returned :code:`TimeSeries` or :code:`TimeSeriesList` objects. By default, this plots the raw
-data points for the last 24 hours.
+data points for the last 24 hours. If there are no data points for the last 24 hours, then plot will throw an exception.
```
Comment (arnekv) [inline docs/source/cognite.rst]: Remove "then" - "...for the last 24 hours, plot will..."

File: docs/source/cognite.rst [hunk 5]
```diff
@@ -13,12 +13,15 @@ You can also pass your API key directly to the CogniteClient.
...
-    >>> c = CogniteClient(api_key="your-key")
+    >>> c = CogniteClient(api_key="<your-api-key>", client_name="your-client-name")
 
 Instantiate a new client
 ------------------------
 Use this code to instantiate a client and get your login status. CDF returns an object with
-attributes that describe which project and service account your API key belongs to.
+attributes that describe which project and service account your API key belongs to. The client_name
+is used to give a unique identifier to the client. You can choose any string for the client_name. The client_name
+can also be provided through the :code: `COGNITE_CLIENT_NAME` environment variable.
```
Comment (arnekv) [inline docs/source/cognite.rst]: Replace "The client_name can also be provided through..." with "You can also provide the client_name through..."

**Pull request merged**